clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -116,9 +116,9 @@ describe("hubConnection", () => {
                     done();
                 });
 
-                const received = [];
+                const received: string[] = [];
                 hubConnection.start().then(() => {
-                    hubConnection.stream("Stream").subscribe({
+                    hubConnection.stream<string>("Stream").subscribe({
```

Comment:
Our other tsconfig files set `noImplicitAny` to `true` which disallows any implicit `any` types (i.e. things the inference engine couldn't infer). This one didn't, but I brought it in line and had to add some explicit type annotations